pkgs/by-name/st/stu/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  stu,
+  testers,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "stu";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "lusingander";
+    repo = "stu";
+    rev = "v${version}";
+    hash = "sha256-T5b3aCepUj8COrKReEaK4JeUbR7Sv7022xSCW8k8Iow=";
+  };
+
+  cargoSha256 = "sha256-DFG/9bnckqLezbitceLtM3CSnKAcQcZlv39VfbkyM/w=";
+
+  passthru.tests.version = testers.testVersion { package = stu; };
+
+  meta = with lib; {
```

Comment:
`with lib;` is an anti-pattern, see here https://github.com/NixOS/nixpkgs/issues/208242.